fitbenchmarking/fitbenchmarking
# Minuit error handling

#### Description of Work

Fixes #337 

#### Testing Instructions

1. Check minuit runs with all data.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
